src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <p>Initial Commit</p>
```

Comment:
Why do you want to merge this into master, this application doesnt appear to do anything?

Reply:
Again, this was just a template - Hamad has further work to commit to it